### PR TITLE
[BEAM-1687] Parallel Python ValidatesRunner Tests

### PR DESCRIPTION
**Pull request by markflyhigh**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

Parallel ValidatesRunner tests to reduce total time of Python postcommit.

Config process=4 since Jenkins build machine has 4 cores.

Python postcommit time reduce to 37mins with this change. [see](https://builds.apache.org/job/beam_PostCommit_Python_Verify/1474/)

### CODE REVIEW

**Comment by markflyhigh** [comment]
Run Python PostCommit

**Comment by markflyhigh** [comment]
Run Python PostCommit

**Comment by markflyhigh** [comment]
+R: @aaltay

File: sdks/python/apache_beam/transforms/ptransform_test.py [hunk 1]
```diff
@@ -45,6 +45,8 @@
 
 
 class PTransformTest(unittest.TestCase):
+  # Enable ValidatesRunner tests running in parallel
```
Comment (aaltay) [inline sdks/python/apache_beam/transforms/ptransform_test.py]: maybe "Enable nose tests running in parallel"

Comment (markflyhigh) [inline sdks/python/apache_beam/transforms/ptransform_test.py]: done

**Comment by markflyhigh** [comment]
I'm not sure if the Jenkins job is running distributed or in one slave. If it's the later case, I think it should be bounded by max number of cores which is four.

@jasonkuster can you provide more details?

**Comment by markflyhigh** [comment]
Also I believe we have limited Jenkins resources (2-3 machines). Having parallel process equals 4 is a pretty reasonable.

**Comment by aaltay** [comment]
That sounds good, thank you. LGTM.

**Pull request closed without merging**